tests/unit/maintainers_test.sh

```diff
@@ -109,16 +112,157 @@ function test_maintainers_main()
   multilineAssertEquals "$CORRECT_TMP_FS_MSG" "$ret"
 
   cd fs || {
-    fail "($LINENO) It was not possible to move to fs directory"
+    fail "(${LINENO}) It was not possible to move to fs directory"
     return
   }
   ret="$(maintainers_main ..)"
   multilineAssertEquals "$CORRECT_TMP_MSG" "$ret"
 
   ret="$(maintainers_main .)"
-  multilineAssertEquals "$CORRECT_TMP_FS_MSG" "$ret"
+  multilineAssertEquals "$ret" "$CORRECT_TMP_FS_MSG"
+
+  cd "$original_dir" || {
+    fail "(${LINENO}) It was not possible to move back from temp directory"
+    return
+  }
```

Comment:
Maybe you can move this `cd` to a specific folder to the `setUp` function and go back to the `tearDown` function; that way, each test can be simplified.